spec/controllers/admin/users_controller_spec.rb

```diff
@@ -18,6 +18,18 @@
       expect(response).to render_template :index
     end
   end
+  describe 'PATCH #togle_confirmation' do
```

Comment:
Although, it passes, I am not sure if it is the right way to test the action. @differentreality kindly review it.